src/lib/extra_index.js

```diff
@@ -3,6 +3,7 @@ import Cytoscape from './components/Cytoscape.react';
 import CytoscapeJS from 'cytoscape';
 import coseBilkent from 'cytoscape-cose-bilkent';
 import cola from 'cytoscape-cola';
+import ctxmenu from 'cytoscape-cxtmenu';
```

Comment:
Seems like `ctx` and `cxt` are used in different contexts. Do they mean the same thing, or is there specific distinctions between `cxt` and `ctx`?